optuna_integration/chainermn.py

```diff
@@ -1,11 +1,10 @@
+from __future__ import annotations
+
 from datetime import datetime
 from typing import Any
 from typing import Callable
-from typing import Dict
-from typing import Optional
 from typing import overload
 from typing import Sequence
-from typing import Tuple
 from typing import Type
```

Comment:
You can alse remove this line.

Reply:
Thank you for your comment. I updated accordingly.

Reply:
Thank you for your comments. I updated accordingly.